src/components/Form/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react'
+import Link from 'next/link'
+
+interface IForm {
```

Comment:
I’m not sure that we should use prefix `I` for interface name. According to this [https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript](url). Please correct me if I’m wrong.